connect/src/spirc.rs

```diff
@@ -270,54 +272,141 @@ impl Spirc {
         Ok((spirc, task.run()))
     }
 
+    /// Safely shutdowns the spirc.
+    ///
+    /// This pauses the playback, disconnects the connect device and
+    /// bring the future initially returned to an end.
+    pub fn shutdown(&self) -> Result<(), Error> {
+        Ok(self.commands.send(SpircCommand::Shutdown)?)
+    }
+
+    /// Resumes the playback
+    ///
+    /// ## Remarks:
```

Comment:
I don't think "Remarks" is an idiomatic Rustdoc paragraph? For me you could just drop the paragraph title and keep the paragraph text underneath it as-is.

Reply:
Hmm, I think I will do some research how to write better rustdoc and adjust the comments accordingly. The "Remarks" originates from a different language where you have separate section and it seems to fit as hint, but it could also be just a paragraph yeah :D

Reply:
I removed the "Remarks" sections and flattened it so that the remarks appear as paragraphs. There doesn't seem to be any real way to do it, besides the example sections which seemed to be pushed quite a bit.